### DISCUSSION TITLE
Configuration stalled

### DISCUSSION THREAD

**Discussion by theScudd** [astrandb/miele#416] [Q&A]
Hello,

I'm attempting to install and configure the integration for the first time. I've made it through what I considered the most challenging steps but I'm stalled when I attempt to allow HA to have access to my appliance data.  Instead of this screen:

![282075295-f2fac53e-cdfc-4df7-a5f1-eacd697a7ba7](https://github.com/astrandb/miele/assets/156456998/70969ff0-6fd6-468c-83da-4fabaae9d6fc)

I get this:

<img width="640" alt="Screenshot 2024-01-15 at 8 53 01 PM" src="https://github.com/astrandb/miele/assets/156456998/8b376250-e1fc-4496-bb28-03e366d150aa">

I followed all of the instructions at this [location](https://www.home-assistant.io/more-info/no-url-available/) to no avail. Any tips?

**Comment by astrandb** [comment]
This is not a problem with the Miele integration itself but something is probably "non-standard" in your local network.

Try this:
-  Find the IP address of your HA instance. Go to Settings->System->Network 
-  Make sure that you can access your HA by entering http://\<your-ip>\:8123 in the browser
-  Setup the integration again. When you get to the "Link account" screen above, you should edit "Your instance URL" to http://\<your-ip\>:8123 and continue with `LINK ACCOUNT`

**Reply by theScudd** [reply]
Thanks for the quick reply. My issue is that I don't get the Link Account screen. When I get to this screen:

<img width="640" alt="Screenshot 2024-01-16 at 7 12 08 AM" src="https://github.com/astrandb/miele/assets/156456998/333f2153-3305-4a2f-b6e2-32af0ae530df">

and click 'Allow', this is the screen I get:

<img width="630" alt="Screenshot 2024-01-16 at 7 13 43 AM" src="https://github.com/astrandb/miele/assets/156456998/1a520132-c14d-4d89-8c35-92f02cb94e5f">

I am able to access HA using http://192.168.0.124:8123.

<img width="625" alt="Screenshot 2024-01-16 at 7 16 48 AM" src="https://github.com/astrandb/miele/assets/156456998/c830df97-1702-41bf-9d09-2802ea9566c4">

I agree that the issue is not the integration, it is my network. Still, I thought someone here may have encountered this before and know of a solution.

**Reply by theScudd** [reply]
Could this be a firewall issue? Do I need to open a port on my router?

**Comment by astrandb** [comment]
Try to connect to your HA with http://192.168.0.124:8123 and do the integration setup from there. You should get the "Link account to..." screen. Press the little pencil icon and enter http://192.168.0.124:8123 and continue. 

Or try another browser or phone/ipad. Once the integration is properly set up, it should work from all browsers.

**Reply by theScudd** [reply]
I tried the setup on my phone. It worked! Not sure what was the root of the problem but I'm happy it is fixed.
Thank you very much for your support and for creating this integration and associated forum. Best wishes!

**Reply by astrandb** [reply]
Good to hear. It is probably a bad cookie left somewhere in your browser. I hope it won't cause you any more trouble.
